packages/constants/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@masknet/constants",
```

Comment:
Did you add it to `config.resolve.alias` in the webpack config? It will speed up the build time (other wise we have to wait for tsc to build out dist files).

Reply:
DONE